src/vivarium_gates_mncnh/data/loader.py

```diff
@@ -227,6 +228,36 @@ def load_lbwsg_exposure(
     return data
 
 
+def load_anc_proportion(
+    key: str, location: str, years: Optional[Union[int, str, list[int]]] = None
+) -> pd.DataFrame:
+    anc_proportion = load_standard_data(key, location, years)
+    year_start, year_end = 2021, 2022
+    lower_value = anc_proportion.loc[(year_start, year_end, "lower_value"), "value"]
+    mean_value = anc_proportion.loc[(year_start, year_end, "mean_value"), "value"]
+    upper_value = anc_proportion.loc[(year_start, year_end, "upper_value"), "value"]
+
+    try:
+        anc_proportion_dist = sampling.get_truncnorm_from_quantiles(
+            mean=mean_value, lower=lower_value, upper=upper_value
+        )
+        num_draws = 1000
+        anc_proportion_draws = anc_proportion_dist.rvs(num_draws).reshape(1, num_draws)
+    except FloatingPointError:
+        print("FloatingPointError encountered, proceeding with caution.")
+        anc_proportion_draws = np.full((1, num_draws), mean_value)
+
+    draw_columns = [f"draw_{i:03d}" for i in range(num_draws)]
```

Comment:
Draw columns need to be the format "draw_{draw_number}" so either "draw_5" or "draw_100"

Reply:
Oh oops, so not draw_005 you mean?